src/dehb/optimizers/dehb.py

```diff
@@ -59,7 +71,19 @@ def __init__(self, cs=None, f=None, dimensions=None, mutation_factor=None,
         # Hyperband related variables
         self.min_budget = min_budget
         self.max_budget = max_budget
-        assert self.max_budget > self.min_budget, "only (Max Budget > Min Budget) supported!"
+        if self.max_budget <= self.min_budget:
+            self.logger.error("Only (Max Budget > Min Budget) is supported for DEHB.")
+            if self.max_budget == self.min_budget:
+                self.logger.error(
+                    "If you have a fixed fidelity, " \
+                    "you can instead run DE as follows: " \
+                    "AsyncDE(cs=configspace, f=target_function, dimensions=" \
+                    f"{self.dimensions}, pop_size={self.dimensions * 2}, " \
+                    f"max_age={self.max_age}, mutation_factor=" \
+                    f"{self.mutation_factor}, crossover_prob={self.crossover_prob}, "\
+                    f"strategy={self.strategy}, budget={self.max_budget}, " \
+                    f"boundary_fix_type={self.fix_type})")
+            sys.exit()
```

Comment:
We should double-check when this happens such that the termination here does not affect the Dask processes.

Reply:
This only happens when the initializer of the DEHBBase class is called, so before any Dask clients are set or initialized. Therefore I don't think this should be a problem. However I thought about rather raising an ```AsssertionError``` to be a little more specific than just exiting. Under the hood this obviously results in the same outcome.